src/ParaTest/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -321,15 +338,14 @@ protected function getDefects(array $defects, $type)
     /**
      * Prints progress for large test collections
      */
-    protected function printProgress()
+    protected function getProgress()
     {
-        printf(
-            ' %' . $this->numTestsWidth . 'd (%3s%%)',
+        return sprintf(
+            ' %' . $this->numTestsWidth . 'd / %' . $this->numTestsWidth . 'd (%3s%%)',
             $this->casesProcessed,
-            floor(($this->casesProcessed / $this->totalCases) * 100)
+            $this->totalCases,
```

Comment:
This is still not reliable.
Given you have skipped / incomplete tests, the total test count decreases. Example:
```
...............................................................  63 / 179 ( 35%)
............................................................... 126 / 136 ( 92%)
..........
```

The total count was originally removed because no total count is better than an incorrect total count.